styleguide/derek/view-templates/stories/_stories.scss

```diff
@@ -216,33 +254,73 @@ $default-bg-img: 'https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.
 }
 
 .storyOverview-itemInner {
-  border: 1px solid $gray-light;
-  padding: 5px;
+  border: 1px solid fade-out($white, .7);
 }
 
 .storyOverview-itemLink {
   @include flex-box;
   @include flex-center;
-  background: $gray-base;
   height: 125px;
+  overflow: hidden;
+  position: relative;
   text-align: center;
   text-decoration: none;
 
   &:hover {
-    background-color: $gray-dark;
+    background-color: fade-out($white, .9);
     text-decoration: none;
+    transition-duration: .2s;
   }
 }
 
+@include storyGridImg('.storyOverview-itemLink-withImg');
```

Comment:
I don't see this mixin being used anywhere else? Do we really need a mixin for this?